### ISSUE TITLE
Bitwarden with Authelia

### ISSUE THREAD

**Issue by robflate** [CVJoint/traefik2#6]
In `ymlfiles/bitwarden.yml` you have the Authelia label. With this on I'm unable to login to BitWarden using the iOS app, it just crashes. If I disable auth, it works. Is this just so people don't unwittingly expose BitWarden or are you able to use Authelia and still login using the BitWarden mobile apps? Thanks.

**Comment by CVJoint** [comment]
I had trouble with that as well. At the time of my last commit I was trying to work on using Traefik labels to bypass API routes, as I'm doing for apps like Sonarr, Radarr, Tautulli, etc. The idea was that I would be required to authenticate with Authelia when I'm using a browser, but mobile apps should bypass the auth and function correctly. What was weird was that my iOS app at the time would actually pull up the Authelia login, so I figured all was good and kept those Traefik labels at the time of that commit. Soon after I had some issues, and figured that a critical thing like bitwarden shouldn't be so locked down, and I wouldn't want to lock any other users out by accident. Sorry for the confusion.

I've just pushed my latest config which has been running great since then. I still use Authelia, but I only apply it to the /admin/ route, and I only enable the /admin/ route when I need to. Hope that helps.
